src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -215,8 +215,10 @@ watch(() => props.pageNumber, function () {
     kTableMountKey.value++
   }
 })
-watch(() => props.headers, function () {
-  kTableMountKey.value++
+watch(() => props.headers, function (val, prev) {
+  if (JSON.stringify(val) !== JSON.stringify(prev)) {
```

Comment:
There's no better way to do object eval than stringify?

Reply:
This is probably the most dependency free/fastest/simplest way (as long as you don't care about key order, which we don't here)

(btw: assuming thats a eval > equal typo?)